zip-0321.rst

```diff
@@ -4,7 +4,243 @@
   Title: Payment Request URIs
   Owners: Kris Nuttycombe (kris@electriccoin.co)
           Daira Hopwood (daira@electriccoin.co)
-  Status: Reserved
+  Status: Proposed
   Category: Wallet
+  Created: 2020-08-28
   Discussions-To: <https://github.com/zcash/zips/issues/347>
   Pull-Request: <https://github.com/zcash/zips/pull/395>
+  License: MIT
+
+
+Terminology
+===========
+
+The key words "MUST", "MUST NOT", "SHOULD", "RECOMMENDED", and "MAY" in this
+document are to be interpreted as described in RFC 2119. [#RFC2119]_
+
+The terms below are to be interpreted as follows:
+
+payment
+  A transfer of funds implemented by a shielded or transparent output of
+  a single Zcash transaction.
+
+
+Abstract
+========
+
+This ZIP proposes a standard format for payment request URIs. Wallets that
+recognize this format enable users to construct payments simply by
+clicking links on webpages or scanning QR codes.
+
+
+Motivation
+==========
+
+In order for a robust transactional ecosystem to evolve for Zcash, it is
+necessary for vendors to be able to issue requests for payment. At present, the
+best option available is to manually specify a payment address, a payment
+amount, and potentially memo field content. Of these three components, existing
+wallets only provide functionality for reading payment addresses in a
+semi-automated fashion. It is then necessary for the user to manually enter
+payment amounts and any associated memo information, which is tedious and may
+be error-prone, particularly if a payment is intended for multiple recipients
+or the memo field information contains structured data that must be faithfully
+reproduced.
+
+This ZIP seeks to eliminate these issues by proposing a standard format that
+wallet vendors may support so that human intervention is required only for
+approval, not creation, of such a payment transaction.
+
+In Bitcoin, two different standards exist to permit vendors to issue payment
+requests that are understood by wallets: BIP-0021 [#bip-0021]_ and BIP-0070
+[#bip-0070]_.  BIP-0021 provides a URI format that can be interpreted by a
+wallet to construct simple, single-recipient transactions; while BIP-0070 uses a
+protobuf-based protocol that permits payment requests to specify the creation
+of transactions of arbitrary complexity.
+
+The format proposed in this ZIP seeks a middle ground between these approaches:
+to provide a URI-based format which supports both the trivial use case and
+the slightly-more-complex situation where a payment may be intended for
+multiple recipients.
+
+Requirements
+============
+
+The format must be a valid URI.
+
+The format must permit the representation of one or more (payment address, amount,
+memo) tuples.
+
+Specification
+=============
+
+The following syntax specification uses ABNF [#RFC5234]_.
+
+URI Syntax:
+
+.. code-block:: EBNF
+
+  zcashurn        = "zcash:" ( zcashaddress [ "?" zcashparams ] / "?" zcashparams )
+  zcashaddress    = 1*base58
```

Comment:
Does this line mean a zcash address begins with a `1` prefix? If so, that's incorrect.

Reply:
No, this means that the `zcashaddress` production is one or more base58 characters. See https://github.com/zcash/zips/pull/395/files#diff-a386636de36a003f2f6c54ee53696106R127. Also you'll note that this production has changed in a later amendment to the PR, and some more qualifications on addresses were added in https://github.com/zcash/zips/commit/8c4e0b5e147edaa9b14415a19e315b5e75368786#diff-a386636de36a003f2f6c54ee53696106R152